libs/accounts/src/lib/accounts-table.tsx

```diff
@@ -137,38 +163,58 @@ export const AccountTable = forwardRef<AgGridReact, AccountTableProps>(
               type="rightAligned"
               cellRenderer={({
                 data,
+                api,
               }: VegaICellRendererParams<AccountFields>) => {
-                return data ? (
-                  <>
-                    <ButtonLink
-                      data-testid="breakdown"
-                      onClick={() => {
-                        setOpenBreakdown(!openBreakdown);
-                        setBreakdown(data.breakdown || null);
-                      }}
-                    >
-                      {t('Breakdown')}
-                    </ButtonLink>
-                    <span className="mx-1" />
-                    <ButtonLink
-                      data-testid="deposit"
-                      onClick={() => {
-                        onClickDeposit && onClickDeposit(data.asset.id);
-                      }}
-                    >
-                      {t('Deposit')}
-                    </ButtonLink>
-                    <span className="mx-1" />
-                    <ButtonLink
-                      data-testid="withdraw"
-                      onClick={() =>
-                        onClickWithdraw && onClickWithdraw(data.asset.id)
-                      }
-                    >
-                      {t('Withdraw')}
-                    </ButtonLink>
-                  </>
-                ) : null;
+                if (!data) return null;
```

Comment:
suggestion (non blocking): remove else keywords here to reduce indentation and improve readability

```
if (!data) return null
if (data.asset.id === marketAssetId ...) {
  // return pinned row ui
}
// return default ui
```